dspace/config/emails/register

```diff
@@ -1,16 +1,16 @@
-# E-mail sent to DSpace users when they register for an account
-#
-# Parameters: {0} is expanded to a special registration URL
-#
-# See org.dspace.core.Email for information on the format of this file.
-#
-Subject: DSpace Account Registration
+## E-mail sent to DSpace users when they register for an account
+##
+## Parameters: {0} is expanded to a special registration URL
+##
+## See org.dspace.core.Email for information on the format of this file.
+##
+#set($subject = "${config.get("dspace.name")} Account Registration")
```

Comment:
I had to change the quotes around `"dspace.name"` to single quotes in order to get it to work. 
Otherwise I got:
```
Exception: Encountered "dspace" at /path/to/config/emails/register[line 7, column 33]
Was expecting one of:
    ")" ...
    <WHITESPACE> ...
    <NEWLINE> ...
    "-" ...
    "+" ...
    "*" ...
    "/" ...
    "%" ...
    <LOGICAL_AND> ...
    <LOGICAL_OR> ...
    <LOGICAL_LT> ...
    <LOGICAL_LE> ...
    <LOGICAL_GT> ...
    <LOGICAL_GE> ...
    <LOGICAL_EQUALS> ...
    <LOGICAL_NOT_EQUALS> ...
```

Reply:
Fixed.